core/src/cubos/core/io/keyboard.cpp

```diff
@@ -0,0 +1,119 @@
+#include <cubos/core/data/deserializer.hpp>
+#include <cubos/core/data/serializer.hpp>
+#include <cubos/core/io/keyboard.hpp>
+
+using cubos::core::data::Deserializer;
+using cubos::core::data::Serializer;
+using cubos::core::io::Key;
+using cubos::core::io::Modifiers;
+using namespace cubos::core;
+
+std::string io::modifiersToString(Modifiers modifiers)
+{
+    std::string result;
+
+    if ((modifiers & Modifiers::Control) != Modifiers::None)
+        result += "C-";
+    if ((modifiers & Modifiers::Shift) != Modifiers::None)
+        result += "S-";
+    if ((modifiers & Modifiers::Alt) != Modifiers::None)
+        result += "M-";
+    if ((modifiers & Modifiers::System) != Modifiers::None)
+        result += "D-";
+
+    return result;
+}
```

Comment:
So modifiers always end in `-`? Wouldn't it make more sense to add the trailing '-' in the `KeyAndModifier`?